build.gradle.kts

```diff
@@ -37,4 +38,20 @@ task("clean") {
 
 buildSrcVersions {
     indent = "    "
+}
+
+detekt {
+    input = files("$projectDir/app/src/main/java")
```

Comment:
How about adding also test files?
Something like: 
`input = files(  
"src/main/java",   
"src/test/java",  
            "src/androidTest/java",
            "src/debug/java",
            "src/release/java",
        )`

Reply:
In my experience, the test files cause more warnings for the way they are written. I know it sounds like an excuse but it happens for example on methods or names.

Reply:
I agree with @sierisimo. I have used static analysis tools with test code before and it really looks bad if your CI pipeline fails because of code smells in your test code.

Reply:
Yes it's harder to keep maintainability on tests, and while I haven't been in a situation where this has been a real problem, I understand your concerns.  LGTM :)